rankers/BiMatchRanker/tests/test_bimatchranker.py

```diff
@@ -137,9 +137,9 @@ def test_bimatchranker_readme():
 
     assert match_score_1 > match_score_2
     assert doc.matches[0].id == '2'
-    assert match_score_1 == pytest.approx(0.9994, 0.001)
+    assert match_score_1 == pytest.approx(0.5333, 0.001)
```

Comment:
why does the value change? it should not be changed by a refactor

Reply:
This is the original value. This can be change with d_miss setting

Reply:
https://github.com/jina-ai/jina-hub/blob/b14a61263a048d18fcbd8ffa4cbe2a6b5f85c5e5/rankers/BiMatchRanker/tests/test_bimatchranker.py#L77
https://github.com/jina-ai/jina-hub/blob/b14a61263a048d18fcbd8ffa4cbe2a6b5f85c5e5/rankers/BiMatchRanker/tests/test_bimatchranker.py#L84

Reply:
Now this is the same as before